src/client/components/footer.js

```diff
@@ -59,6 +59,12 @@ function Footer(props) {
 						</small>
 					</Col>
 					<Col className="text-right" xs={4}>
+						<small>
+							<a href="/admin-logs">
```

Comment:
Yes please.
I would also like to see this link duplicated in the "privileges" dropdown in the menu bar for easy access (considering this will eventually only be visible to a limited amount of users and is quite a utilitarian menu item, I think the duplication is fine in this case.